website/content/docs/concepts/secure-variables.mdx

```diff
@@ -0,0 +1,124 @@
+data like batch job results. These should be stored in a separate
+database. Secure Variables are also not intended to be a full replacement for
+HashiCorp Vault. If you need powerful options like dynamic secrets or transit
+encryption, you'll need to use Vault.
+
+## ACL for Secure Variables
+
+Every Secure Variable belongs to a specific Nomad namespace. ACL policies can
+restrict access to Secure Variables within a namespace on a per-path basis,
+using a list of `path` blocks, under the `namespace.secure_variables` block. See
+the [ACL policy specification] docs for details about the syntax and structure
+of an ACL policy.
+
+Path definitions may also include wildcard symbols, also called globs, allowing
+a single path policy definition to apply to a set of paths within that
+namespace. For example, the policy below allows full access to secure variables
+at all paths in the "dev" namespace that are prefixed with "project/", but only
+read access to paths prefixed with "system/". Note that the glob can match an
+empty string but all other characters are strictly matched. This policy grants
+read access to paths prefixed with "system/" but not a path named "system"
+(without a trailing slash).
+
+```hcl
+namespace "dev" {
+  policy       = "write"
+  capabilities = ["alloc-node-exec"]
+
+  secure_variables {
+
+    # full access to secrets in all "project" paths
+    path "project/*" {
+      capabilities = ["write", "read", "destroy", "list"]
+    }
+
+    # read/list access within a "system/" path belonging to administrators
+    path "system/*" {
+      capabilities = ["read"]
+    }
+  }
+}
+```
+
+The available capabilities for Secure Variables are as follows:
+
+| Capability | Notes                                                                                                                        |
+|------------|------------------------------------------------------------------------------------------------------------------------------|
+| write      | Create or update Secure Variables at this path. Includes the "list" capability but not the "read" or "destroy" capabilities. |
+| read       | Read the decrypted contents of Secure Variables at this path. Also includes the "list" capability                            |
+| list       | List the metadata but not contents of Secure Variables at this path.                                                         |
+| destroy    | Delete Secure Variables at this path.                                                                                        |
+
+## Task Access to Secure Variables
+
+In Nomad 1.4.0 tasks can access Secure Variables only with the [`template`]
+block. The [workload identity] for each task grants it automatic access to
+Secure Variables found at Nomad-owned paths with the prefix `nomad/jobs/`,
+followed by the job ID, task group name, and task name.
+
+```
+nomad/jobs/$job_id/$task_group/$task_name
+nomad/jobs/$job_id/$task_group
+nomad/jobs/$job_id
+nomad/jobs
+```
```

Comment:
I _think_ this might be better expressed as a literal ACL policy like the one above - e.g. it's not clear what capabilities my task automatically has, (and are they different at each level?)

Reply:
I love that idea. Will do.